docs/configuration/elasticsearch.md

```diff
@@ -0,0 +1,22 @@
+# Elasticsearch
+
+[Elasticsearch](https://www.elastic.co/) is the native search engine and it must be configured since Magento 2.4 onwards.
```

Comment:
```suggestion

Warden supports use of Elasticsearch. It is enabled by default on some environments such as `magento2` where it is a baseline requirement for the application, but it can be enabled and used on other environments types such as `shopwere` and `symfony` as an optional service. Please refer to specific installation guides for configuration specific to a given application.

[Elasticsearch](https://www.elastic.co/) is a leading free and open search and analytics solution built on Apache Lucene.
```